include/spirv-tools/optimizer.hpp

```diff
@@ -907,6 +907,8 @@ Optimizer::PassToken CreateConvertToSampledImagePass(
 // from every function in the module.  This is useful if you want the inliner to
 // inline these functions some reason.
 Optimizer::PassToken CreateRemoveDontInlinePass();
+
+Optimizer::PassToken CreateVarForFuncCallPass();
```

Comment:
The name of the pass is confusing .. typically the pass name starts with a verb. "variable for function call" .. I cannot understand what it means. In addition, could you please add a comment here to explain this pass?